scripts/pathfinders_validation.py

```diff
@@ -0,0 +1,439 @@
+                "Likelihood score": pa.Column(str, pa.Check.isin(PFEnums.RISK_SCORES)),
+                "Impact score": pa.Column(str, pa.Check.isin(PFEnums.RISK_SCORES)),
+                "Mitigations": pa.Column(str, pa.Check.word_count_less_than(100)),
+                "Risk Owner": pa.Column(str),
+            },
+            row_idxs_to_drop=[0],
+            drop_empty_rows=True,
+            messages={
+                **messages,
+                "word_count_less_than": "Please enter a maximum of 100 words.",
+            },
+        ),
+        "Underspend": ta.TableSchema(
+            id_tag="PF-TABLE-ID-14",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "What's your underspend for this financial year?": pa.Column(float),
+            },
+            row_idxs_to_drop=[0],
+            messages=messages,
+        ),
+        "Current underspend": ta.TableSchema(
+            id_tag="PF-TABLE-ID-15",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "What's your current underspend for this financial year?": pa.Column(float),
+            },
+            row_idxs_to_drop=[0],
+            messages=messages,
+        ),
+        "Underspend requested": ta.TableSchema(
+            id_tag="PF-TABLE-ID-16",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "How much underspend are you asking for with a credible plan?": pa.Column(float),
+            },
+            row_idxs_to_drop=[0],
+            messages=messages,
+        ),
+        "Spending plan": ta.TableSchema(
+            id_tag="PF-TABLE-ID-17",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "How do you plan to spend this value in the next financial year?": pa.Column(
+                    str, pa.Check.word_count_less_than(500)
+                ),
+            },
+            row_idxs_to_drop=[0],
+            messages={
+                **messages,
+                "word_count_less_than": "Please enter a maximum of 500 words.",
+            },
+        ),
+        "Forecast spend": ta.TableSchema(
+            id_tag="PF-TABLE-ID-18",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "What is your forecast spend for the next financial year?": pa.Column(float),
+            },
+            row_idxs_to_drop=[0],
+            messages=messages,
+        ),
+        "Forecast and actual spend": ta.TableSchema(
+            id_tag="PF-TABLE-ID-19",
+            worksheet_name="Finances",
+            section="Portfolio level finances",
+            columns={
+                "Type of spend": pa.Column(str),  # TODO: isin
```

Comment:
Type of spend should be unique but if the constraint is broken then it's a spreadsheet configuration error rather than a user error because the types of spend are pre-defined and not user entered. In other words, if this constraint was broken then they must have tampered with the spreadsheet, which is very unlikely. If they had though, I'm not sure if a user error or an exception would be the best way to handle it. I think as it would be such a special case I can see the benefit in it throwing an exception during later validation and then handling this case via a support request.